servers/lib/package.json

```diff
@@ -34,6 +34,7 @@
     "express": "^4.18.2",
     "graphql": "^16.8.1",
     "mock-fs": "^5.2.0",
+    "react": "^18.2.0",
```

Comment:
Is this package used for tests?

Reply:
Yes it is required by Apollo

Reply:
Please move it to devDependencies If it is only for the tests.